storage/store.go

```diff
@@ -1195,8 +1197,11 @@ func (s *Store) resolveWriteIntentError(wiErr *proto.WriteIntentError, rng *Rang
 		}
 		bArgs.Add(pushArgs)
 	}
+	b := &client.Batch{}
+	b.InternalAddCall(client.Call{Args: bArgs, Reply: bReply})
```

Comment:
No, this is just the first instance. It will be used everywhere an `proto.Internal*` request occurs and in internal uses of `proto.EndTransaction` which use a commit trigger.